app/konnect/runtime-manager/runtime-instances/gateway-runtime-docker.md

```diff
@@ -70,48 +70,48 @@ Use the following `docker run` command sample as a guide to compile your actual
 {% navtabs codeblock %}
 {% navtab Any Unix shell %}
```

Comment:
Just a small nit pick, but it looks like Unix and Windows are the same, so I though maybe we could just have the script and get rid of the tabs.